src/types.ts

```diff
@@ -0,0 +1 @@
+export * from '@square/web-payments-sdk-types';
```

Comment:
borrowing this [types.ts](https://github.com/snowpackjs/snowpack/blob/5d700e2dc7b7a6e4703630a7178fd993f40ecb5e/snowpack/src/types.ts) pattern from snowpack where they are [also exporting types](https://github.com/snowpackjs/snowpack/blob/5d700e2dc7b7a6e4703630a7178fd993f40ecb5e/snowpack/src/index.ts#L15)